data/profile.d/fedora-kinoite.conf

```diff
@@ -9,3 +9,11 @@ base_profile = fedora-kde
 # Match os-release values.
 os_id = fedora
 variant_id = kinoite
+
+[Storage]
+default_scheme = BTRFS
+btrfs_compression = zstd:1
+default_partitioning =
```

Comment:
You are redefining `default_partitioning` from `anaconda.conf`. It looks like this:

```
default_partitioning =
    /     (min 1 GiB, max 70 GiB)
    /home (min 500 MiB, free 50 GiB)
```

I guess you want to keep the previous settings, so `fedora-kinoite` and `fedora-silverblue` should probably define something like this:

```
default_partitioning =
    /     (min 1 GiB, max 70 GiB)
    /home (min 500 MiB, free 50 GiB)
    /var  (btrfs)
```

Reply:
Thinking of future usage: is it possible for `/` and `/home` to have no attribute, resulting in the min/max/free values being inherited from anaconda.conf? That way any updates to min/max/free values in anaconda.conf just past through, and we don't have derivatives ending up with stale settings?

Reply:
> Thinking of future usage: is it possible for `/` and `/home` to have no attribute, resulting in the min/max/free values being inherited from anaconda.conf?

We have no control over that. We just generate a list of configuration files and give this list to `configparser`. The inheritance is very primitive and we would like to keep it that way.

> That way any updates to min/max/free values in anaconda.conf just past through, and we don't have derivatives ending up with stale settings?

We use unit tests to check this kind of dependencies between profiles.